UserBundle/DependencyInjection/OpenOrchestraUserExtension.php

```diff
@@ -29,6 +29,7 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.yml');
         $loader->load('form.yml');
         $loader->load('display.yml');
+        $loader->load('temporary_fix.yml');
```

Comment:
You should name it with a more accurate name indicating what is fixed, it will be easier to think about removing it